components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -1329,7 +1330,8 @@ constexpr char kOptimismSwapBaseAPIURL[] = "https://optimism.api.0x.org/";
 constexpr char kArbitrumSwapBaseAPIURL[] = "https://arbitrum.api.0x.org/";
 constexpr char kBaseSwapBaseAPIURL[] = "https://base.api.0x.org/";
 constexpr char kSwapBaseAPIURL[] = "https://api.0x.org/";
-constexpr char kBuyTokenPercentageFee[] = "0.00875";
+constexpr double k0xBuyTokenFeePercentage = 0.875;
+constexpr double k0xProtocolFeePercentage = 0.15;
```

Comment:
Is it ok to name it as percentage? Fraction maybe?

Reply:
I used percentage because it values are `0.875%` and `0.15%`. If we name it fraction, the values should be `0.00875` and `0.0015` respectively.